packages/ui/src/components/pages/Accounts/AccountControls/index.tsx

```diff
@@ -9,13 +9,19 @@ import { EventName, triggerEvent } from 'utils/eventemitter';
 enum AccountControlsAction {
   RenameAccount = 'Rename',
   RemoveAccount = 'Remove',
+  ShowAddressQrCode = 'Show QR Code',
 }
 
 const AccountControlsOptions = [
   {
     action: AccountControlsAction.RenameAccount,
     event: EventName.OpenRenameAccountDialog,
   },
+  {
+    action: AccountControlsAction.ShowAddressQrCode,
+    event: EventName.OpenShowAddressQrCodeDialog,
+    className: 'min-[301px]:hidden',
+  },
```

Comment:
We would want to make this the first option since I think it might be more frequently used than rename or remove. Nice work on the className.